README.md

```diff
@@ -91,6 +91,54 @@ cd build-osp-magnum/Release
 
 If you just want to test out the project so far, then see the [Actions](https://github.com/TheOpenSpaceProgram/osp-magnum/actions) tab on GitHub to obtain automated builds for Linux or Windows.
 
+## Controls
+The most important thing to remember, the controls!
```

Comment:
maybe "they will be rather familiar if you have played Kerbal Space Program".

Reply:
That seems reasonable.

Reply:
I'm not sure, I don't want to get into the habit of talking about KSP in the documentation as it might come off to users that we are trying to be a KSP clone

I don't mind the change so I'll add it, just something to think about